components/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,223 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import styles from './InviteAdminModal.module.css';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import Icon from "../../assets/Icon";
+
+export default function InviteAdminModal() {
+  const [open, setOpen] = React.useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const [state, setState] = React.useState({
+    name1: "",
+    email1: "",
+    name2: "",
+    email2: "",
+    name3: "",
+    email3: "",
+  })
+
+  function handleChange(evt) {
+    const value = evt.target.value;
+    setState({
+      ...state,
+      [evt.target.name]: value
+    });
+  }
+
+  const theme = createTheme({
+    palette: {
+      primary: {
+        main: '#253C85',
+      },
+    },
+  });
+
+  // const adminRow = (num) => {
+  //   return (
+  //     <div className={styles['row']}>
+  //       <div className={styles['col']}>
+  //         <TextField
+  //           name="name1"
+  //           value={state.name1}
+  //           onChange={handleChange}
+  //           fullWidth
+  //           autoFocus
+  //           margin="dense"
+  //           id="name"
+  //           placeholder="Firstname Lastname"
+  //           type="text"
+  //           variant="standard"
+  //           className={styles['font']}
+  //         />
+  //       </div>
+  //       <div className={styles['col']}>
+  //         <TextField
+  //           name="email1"
+  //           value={state.email1}
+  //           onChange={handleChange}
+  //           fullWidth
+  //           autoFocus
+  //           margin="dense"
+  //           id="name"
+  //           placeholder="thisisanemail@email.com"
+  //           type="text"
+  //           variant="standard"
+  //           className={styles['font']}
+  //         />
+  //       </div>
+  //     </div>
+  //   )
+  // }
+
+  return (
+    <div>
+      <ThemeProvider theme={theme}>
```

Comment:
Nice use of the ThemeProvider! We will be making a global MUI theme soon, I'll sync up with you on that and how it could change your code, but it shouldn't require much changes.